.github/workflows/CI.yml

```diff
@@ -112,14 +112,15 @@ jobs:
         with:
           name: test-reports
           path: |
-            **/build/reports/tests/
+            **/reports/jacoco/mergedReport/
```

Comment:
By updating this line it removed the junit test reports that include logging alongside the test class which I've always found really useful for debugging.

Or if you'd like seperate out the artifacts so there is one for the project reports (junit) and another for the project-wide reports (the merged jacoco report)

Reply:
I'll keep both